From a91166ad6e57e6658cf76b36262f33ab6d980a0c Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:47 -0400
Subject: [tools] perf: Add 'build-test' make target

Message-id: <1408953210-26343-280-git-send-email-jolsa@redhat.com>
Patchwork-id: 89030
O-Subject: [PATCH RHEL7.1 BZ1133083 279/322] perf tools: Add 'build-test' make target
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit a7077234d240f16ead29ffeb3cc17ceada50ea5f
Author: Namhyung Kim <namhyung@kernel.org>
Date: Thu Jan 16 10:52:53 2014 +0900

description
===========
Currently various build test can be performed using a Makefile named
tests/make, so one needs to remember and specify it with -f option on
command line.

Add the 'build-test' target in the main Makefile as a shortcut.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Makefile b/tools/perf/Makefile
index eefb9fb..cb2e586 100644
--- a/tools/perf/Makefile
+++ b/tools/perf/Makefile
@@ -75,6 +75,12 @@ clean:
  $(make)
 
 #
+# The build-test target is not really parallel, don't print the jobs info:
+#
+build-test:
+ @$(MAKE) -f tests/make --no-print-directory
+
+#
 # All other targets get passed through:
 #
 %:
-- 
1.7.1